Change autoload to PSR-4

odan opened the pull request:
Fixes #217

odan committed:
Change autoload to PSR-4 #217

odan committed:
Change autoload to PSR-4 #217

lulco commented:
Thanks for contribution! Any classnames need change? I’m just not sure if it isn’t breaking change in some cases. That’s why it is in 2.0.0 milestone. I have to check it :)

odan committed:
Move namespace Dumper to Phoenix\Dumper

odan committed:
Move namespace Dumper to Phoenix\Dumper

odan committed:
Fix deprecated test class name

odan committed:
Fix deprecated test class name

odan committed:
Move namespace Comparator to Phoenix\Comparator

odan committed:
Move namespace Comparator to Phoenix\Comparator

odan committed:
Move namespaces Phoenix namespace

odan committed:
Move namespaces Phoenix namespace

odan committed:
Fix tests

odan committed:
Fix tests

odan committed:
Fix tests

odan committed:
Fix tests

odan commented:
```
Changed composer.json: Done
Moved Namespaces: Done

MySql Tests: OK
PG Tests: Not tested

Linter check: OK
PHP 8.0 compatibility check: OK
PHP 8.1 compatibility check: OK
PHPStan Level 4: OK (No Errors)
```

odan committed:
Update changelog and readme

odan committed:
Update changelog and readme

lulco committed:
Trying to run tests on forked PRs

lulco committed:
Fixed PR tests (#224)

odan committed:
Merge remote-tracking branch 'origin/psr4' into psr4
